app/state_machines/allocation_state_machine.rb

```diff
@@ -0,0 +1,11 @@
+class AllocationStateMachine < FiniteMachine::Definition
+  initial :unfilled
+
+  event :fill, %i[none unfilled filled] => :filled
```

Comment:
what is the `none` state and how do you get to it? The initial state is `unfilled` and none of the events will transition to `none` ?

Reply:
Since `status` currently allows `nil` values as it was introduced a while back, we should allow the allocation to exist in this `none` state (https://github.com/piotrmurach/finite_machine#24-initial) until we update all values (with the refresh rake task) in the environments. After that happens we can get rid of the `none` state and only have the 3

Reply:
👍 thanks for explaining

Reply:
Actually on revisiting this I ended up removing it as it is there by default, and we don't have to explicitly define it